covalent_awslambda_plugin/exec.py

```diff
@@ -0,0 +1,36 @@
+import os
+
+import boto3
+import cloudpickle as pickle
+
+
+def handler(event, context):
+    os.environ["HOME"] = "/tmp"
+    os.chdir("/tmp")
+
+    s3_bucket = os.environ["S3_BUCKET_NAME"]
+    if not s3_bucket:
+        raise ValueError("Environment variable S3_BUCKET_NAME was not found")
+
+    func_filename = os.environ["COVALENT_TASK_FUNC_FILENAME"]
```

Comment:
I don't think this will work. Lambda does not let you customize these environment variables, and in any case the same environment could be used for multiple tasks. Task-specific metadata will be passed to the Lambda handler in the event payload.

Reply:
ah so rather than using env vars... is the idea to pass these required values as the payload in the `event`?

Reply:
Yeah, see #46.

Reply:
updated!